tensorflow/contrib/tensorboard/db/loader.cc

```diff
@@ -112,8 +112,10 @@ int main(int argc, char* argv[]) {
   }
   uint64 elapsed = env->NowMicros() - start;
   LOG(INFO) << "Loaded " << AddCommas(offset) << " bytes with "
-            << AddCommas(records) << " records at "
-            << AddCommas(offset / (elapsed / 1000000)) << " bps";
+            << AddCommas(records) << " records";
+  if (elapsed > 0) {
+    LOG(INFO) << "bps=" << (uint64)(offset / (elapsed / 1000000.0));
```

Comment:
This creates an additional log line. Maybe we could have a ternary instead like `"bps=" << (elapsed == 0 ? 0 : (uint64)(offset / (elapsed / 1000000.0)))`.

Reply:
Thank you for the review. I moved the bps to one log line like before and used the ternary as you suggest.

I added a condition to report the bps only if elapsed is greater than 0  because it did not seem right to say bps of 0 if 'n' bytes were read and elapsed was 0. 

How about I do this: 
(elapsed == 0 ? offset : static_cast<uint64>(offset / (elapsed / 1000000.0)))

So if elapsed is 0, we will report bps as the bytes read (ie offset) as that many bytes were processed in a sec. 

————
The log line will look like as before: 
2018-03-28 21:02:17.282332: I tensorflow/contrib/tensorboard/db/loader.cc:114] Loaded 590,308 bytes with 29 records at 1907172 bps